### PR TITLE
validator: pod spec changes

### PR DESCRIPTION
**Pull request by eyakubovich**

### CODE REVIEW

File: ace/validator.go [hunk 1]
```diff
@@ -303,18 +303,15 @@ func validateContainerAnnotations(metadataURL string, pm *schema.PodManifest) re
 	return r
 }
 
-func validateContainerMetadata(metadataURL string, pm *schema.PodManifest) results {
+func validatePodMetadata(metadataURL string, pm *schema.PodManifest) results {
 	r := results{}
 
-	uid, err := metadataGet(metadataURL, "/pod/uid")
+	_, err := metadataGet(metadataURL, "/pod/uuid")
```
Comment (jonboulle) [inline ace/validator.go]: can you actually validate what is returned here?

Comment (eyakubovich) [inline ace/validator.go]: I can't now. Since UUID is not part of manifest there's no other way to know uuid

Comment (jonboulle) [inline ace/validator.go]: Validate that it is a valid UUID at least is what I mean.

File: ace/validator.go [hunk 2]
```diff
@@ -303,18 +303,20 @@ func validateContainerAnnotations(metadataURL string, pm *schema.PodManifest) re
...
+	uuid, err := metadataGet(metadataURL, "/pod/uuid")
 	if err != nil {
 		return append(r, err)
 	}
-	if strings.ToLower(string(uid)) != strings.ToLower(pm.UUID.String()) {
-		return append(r, fmt.Errorf("UUID mismatch: %v vs %v", string(uid), pm.UUID.String()))
+
+	_, err = types.NewUUID(string(uuid))
+	if err != nil {
+		return append(r, fmt.Errorf("invalid UUID returned: %v", err))
```
Comment (jonboulle) [inline ace/validator.go]: maybe print the invalid UUID??

**Comment by jonboulle** [comment]
One nit but LGTM

**Comment by jonboulle** [comment]
thank you!!

File: ace/validator.go (hunk 2 above)
Comment (eyakubovich) [inline ace/validator.go]: yeah, looks like NewUUID error is not going to.

**Comment by jonboulle** [comment]
gentleman and a saint

**Pull request merged**